web-animations-2/Overview.bs

```diff
@@ -969,6 +972,30 @@ Add the following sentence:
 
 </div>
 
+### Calculating the progress of an animation ### {#the-progress-of-an-animation}
+
+  The [=animation/progress=] of an [=animation=] is calculated as follows:
+
+  <dl class=switch>
+
+  : If <em>any</em> of the following are true:
+    * the animation does not have an [=animation/associated effect=], or
+    * the animation's associated effect's [=end time=] is infinite, or
+    * the animation's associated effect's [=end time=] is zero, or
```

Comment:
I think we can replace "the animation's associated effect's [=end time=]" with "the animation's [=associated effect end=]".

Reply:
That sounds goo. Done.